### PR TITLE
Add Instructure provider.

### PR DESCRIPTION
**Pull request by frankperez87**
Provider: https://www.instructure.com/
API Docs: https://canvas.instructure.com/doc/api/file.oauth.html

Instructure allows for hosted instances per customer so I added a new configuration to allow the setting of an `instance_url` within the service's config.

### CODE REVIEW

**Comment by pochocho** [comment]
This provider would be a great addition

**Review by lucasmichot — APPROVED** [review]
Thanks for the contribution, GTM

**Review by mdaoustHL — APPROVED** [review]
🎉

File: src/Instructure/composer.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    ],
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Frank Perez",
+            "email": "frank@fjp.io"
+        }
+    ],
+    "require": {
+        "php": "^7.2 || ^8.0",
```
Comment (atymic) [inline src/Instructure/composer.json:19]:
```suggestion
        "php": "^7.4 || ^8.0",
```

**Review by atymic — CHANGES_REQUESTED** [review]

**Comment by atymic** [comment]
Looks good to me!

**Review by atymic — APPROVED** [review]

**Pull request merged**